### PR TITLE
Added support for project name

### PR DESCRIPTION
**Pull request by zeh**
Thanks for this WakaTime extension. However, I felt like it was missing the project name - my VSC projects would always be reported as "Unknown Project".

With this small change, the plugin now also reports the `--project` name to the command line interface. The project name is inferred from the workspace -- that is, the current folder name is used. This is consistent with the workspace name VSCode shows in its title window.

In other words, this box in WakaTime's dashboard...

image

...now becomes this:

image

Your project names, however, may vary.

### CODE REVIEW

**Comment by alanhamlett** [comment]
Thanks!

**Comment by alanhamlett** [comment]
Released in v1.0.6 of the plugin.

**Comment by zeh** [comment]
Thanks for the quick review @alanhamlett!
